docs/breaking-changes.md

```diff
@@ -12,6 +12,15 @@ This document uses the following convention to categorize breaking changes:
 * **Deprecated:** An API was marked as deprecated. The API will continue to function, but will emit a deprecation warning, and will be removed in a future release.
 * **Removed:** An API or feature was removed, and is no longer supported by Electron.
 
+## Planned Breaking API Changes (31.0)
+
+### Behavior Changed: `nativeImage.toDataURL` will preseve PNG colorspace
```

Comment:
I went for the behavior that matches the upstream change, we can avoid a breaking change by reverting the colorspace change on our side but based on the upstream issue seems like decoder and encoder will switch to use skia eventually, so maybe this is the right change. Open to discussion.

```diff
diff --git a/shell/common/skia_util.cc b/shell/common/skia_util.cc
index bace4213b8..0190cadc2c 100644
--- a/shell/common/skia_util.cc
+++ b/shell/common/skia_util.cc
@@ -64,6 +64,7 @@ bool AddImageSkiaRepFromPNG(gfx::ImageSkia* image,
   if (!gfx::PNGCodec::Decode(data, size, &bitmap))
     return false;
 
+  bitmap.setColorSpace(nullptr);
   image->AddRepresentation(gfx::ImageSkiaRep(bitmap, scale_factor));
   return true;
 }
```

Reply:
I think its fine to have a breaking change